### PR TITLE
Migration to AWS S3

### PR DESCRIPTION
**Pull request by shd101wyy**

### CODE REVIEW

**Comment by shd101wyy** [comment]
@dwightguth I finished changing the pom.xml. Also the newest version of rv-install is deployed to s3 already.

**Comment by shd101wyy** [comment]
@dwightguth Wait NVM I made a mistake. Don't review yet.

**Comment by shd101wyy** [comment]
@dwightguth Done. You can review this one now. Thank you.

**Comment by dwightguth** [comment]
This seems really weird. You went to the trouble of creating a pom for the project only to write scripts that invoke deploy-file. You should pick one or the other. Either make the project a maven project and build and deploy by running mvn deploy, or else don't create a pow

**Comment by shd101wyy** [comment]
@dwightguth This project was an ant project. I actually don't know how to change it to maven project. The `pom.xml` is only used to install the extension `aws-maven`.

**Comment by dwightguth** [comment]
Let's talk tomorrow about this. I see no point in changing the build system halfway

**Comment by shd101wyy** [comment]
@dwightguth Done.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 bin
```
Comment (dwightguth) [inline .gitignore]: can you delete these three lines from the .gitignore now that they're no longer needed?

**Comment by shd101wyy** [comment]
@dwightguth done.

**Pull request merged**